res/values/strings.xml

```diff
@@ -771,7 +770,7 @@
     <string name="contact_request_title_desktop">Contact request</string>
     <string name="delete_message_desktop">Delete message</string>
     <string name="more_info_desktop">More info</string>
-    <string name="logout_desktop">Logout</string>
+    <string name="logout_desktop">Log out</string>
     <string name="timestamp_format_m_desktop">MMM D</string>
```

Comment:
Is this correct?

Reply:
From wikipedia seem correct "Logout", because: Logout is the English term to indicate the exit procedure from a computer system or computer application. It comes from the contraction of *log out*, exit from the log, the chronological log of events typical of any information system.

Reply:
Here it is a direct action rather than a noun, is what I gathered.
Getting to the logout, vs. logging out. You don't "logout", you log out.

Reply:
Ok

Reply:
Oh, my question was actually about the "MMM D" Why isn't it "MM DD"?

Reply:
indeed, there is a translator comment missing. i assume it is some sort of date formatting, where `MMM D` expands to `Jul 15` where `MM DD` expands to `07 15`.